oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/continuous/ContinuousProfilingQueryService.java

```diff
@@ -76,10 +119,118 @@ public List<ContinuousProfilingPolicyTarget> queryContinuousProfilingServiceTarg
+                    final ContinuousProfilingMonitoringProcess process = new ContinuousProfilingMonitoringProcess();
+                    process.setId(p.getId());
+                    process.setName(p.getName());
+                    process.setDetectType(p.getDetectType());
+                    process.setLabels(p.getLabels());
+
+                    final EBPFProfilingTaskSummary processSummaryItem = processSummary.get(p.getId());
+                    if (processSummaryItem != null) {
+                        process.setTriggeredCount(processSummaryItem.getCount());
+                        process.setLastTriggerTimestamp(processSummaryItem.getLastTriggerTime());
+                    }
+
+                    return process;
+                }).collect(Collectors.toList()));
+            return result;
+        }).collect(Collectors.toList());
+    }
+
+    private <T> Map<T, EBPFProfilingTaskSummary> buildSummaryByKey(List<EBPFProfilingTaskRecord> records, Function<EBPFProfilingTaskRecord, T> groupBy) {
+        return records.stream().collect(Collectors.groupingByConcurrent(groupBy, buildSummaryCollector()));
+    }
+
+    private List<EBPFProfilingTaskRecord> queryRecentTriggeredTasks(String serviceId, Collection<ContinuousProfilingTargetType> targets) throws IOException {
+        final Calendar timeInstance = Calendar.getInstance();
+        timeInstance.add(Calendar.HOUR, -RECENT_TRIGGERED_HOURS);
+        return getEbpfProfilingTaskDAO().queryTasksByTargets(serviceId, null,
+            targets.stream().map(EBPFProfilingTargetType::valueOf).collect(Collectors.toList()),
+            EBPFProfilingTriggerType.CONTINUOUS_PROFILING, timeInstance.getTimeInMillis(), 0);
+    }
+
+    /**
+     * Summary all records to one summary
+     */
+    private Collector<EBPFProfilingTaskRecord, EBPFProfilingTaskSummary, EBPFProfilingTaskSummary> buildSummaryCollector() {
+        return Collector.of(EBPFProfilingTaskSummary::new,
+            (result, task) -> {
+                result.setCount(result.getCount() + 1);
+                if (task.getStartTime() > result.getLastTriggerTime()) {
+                    result.setLastTriggerTime(task.getStartTime());
+                }
+                result.getRecords().add(task);
+            },
+            (result1, result2) -> {
+                result1.setCount(result1.getCount() + result2.getCount());
+                if (result2.getLastTriggerTime() > result1.getLastTriggerTime()) {
+                    result1.setLastTriggerTime(result2.getLastTriggerTime());
+                }
+                result1.getRecords().addAll(result2.getRecords());
+                return result1;
+            });
+    }
+
+    @Data
+    private static class EBPFProfilingTaskSummary {
+        // count of triggered tasks
+        private int count;
+        // last trigger time
+        private long lastTriggerTime;
+        // all triggered tasks
+        private List<EBPFProfilingTaskRecord> records;
```

Comment:
I think in the summary page, we don't need this. Don't we expose the query for details separately?